src/server/stream_family.cc

```diff
@@ -287,17 +287,31 @@ OpResult<vector<GroupInfo>> OpListGroups(const DbContext& db_cntx, string_view k
   return result;
 }
 
+constexpr int8_t create_group_opts_mkstream = 1 << 0;
+
 struct CreateOpts {
   string_view gname;
   string_view id;
+  int8_t flags = 0;
```

Comment:
uint8_t

Reply:
Done. Thanks for the quick review and suggestion.

Reply:
Thanks for doing this so quickly! Looking forward to talking to you next week! 👍🏼